### PR TITLE
Admin nav bar

### PR DESCRIPTION
**Pull request by Yvonne-Wong**
Created a navbar that displays for admins that includes a link to logout page. logout page now has a button to redirect to homepage. :)

### CODE REVIEW

File: Areas/Identity/Data/WhaleSpottingDbContext.cs [hunk 1]
```diff
@@ -0,0 +1,26 @@
+using System;
```
Comment (ChristianMcc) [inline Areas/Identity/Data/WhaleSpottingDbContext.cs]: In my mind this was part of the last pull request, and either this wasn't needed or something similar has already been merged

File: Areas/Identity/Pages/_ValidationScriptsPartial.cshtml [hunk 2]
```diff
@@ -0,0 +1,18 @@
+<environment include="Development">
```
Comment (ChristianMcc) [inline Areas/Identity/Pages/_ValidationScriptsPartial.cshtml:1]: Likewise I'm surprised this file is only now being added as part of this PR

File: appsettings.json [hunk 3]
```diff
@@ -13,5 +13,8 @@
       }
     }
   },
-  "AllowedHosts": "*"
+  "AllowedHosts": "*",
+  "ConnectionStrings": {
+    "WhaleSpottingDbContextConnection": "Server=(localdb)\\mssqllocaldb;Database=whale-spotting;Trusted_Connection=True;MultipleActiveResultSets=true"
```
Comment (ChristianMcc) [inline appsettings.json]: we can delete these changes (this isn't our connection string anyway)

**Review by ChristianMcc — COMMENTED** [review]
Few questions about unnecessary files but looking good! Do you know how well it's working with logging in/out at the moment? does it update instantly?

File: Areas/Identity/Data/WhaleSpottingDbContext.cs (hunk 1 above)
Comment (Yvonne-Wong) [reply Areas/Identity/Data/WhaleSpottingDbContext.cs]: I scaffolded the app again yesterday to get a copy of the logout files. So we gained some extra files again. The ValidationScriptsPartial was generated alongside the the logout files.  :)

File: Areas/Identity/Pages/_ValidationScriptsPartial.cshtml (hunk 2 above)
Comment (Yvonne-Wong) [reply Areas/Identity/Pages/_ValidationScriptsPartial.cshtml:1]: Please see above comment :)

File: appsettings.json (hunk 3 above)
Comment (Yvonne-Wong) [reply appsettings.json]: Actioned :)

File: Areas/Identity/Data/WhaleSpottingDbContext.cs (hunk 1 above)
Comment (chloejonessoftwire) [reply Areas/Identity/Data/WhaleSpottingDbContext.cs]: For this file in particular it is exactly the same as the old one, think it's been deleted and regenerated upon scaffolding again, that's why it appears to be 'new' :)

**Review by ChristianMcc — APPROVED** [review]
LGTM, who knows what the rerendering problem was and how to solve it 😂

**Pull request merged**